### PR TITLE
Add trim_galore (tested) and updated conda env (untested)

### PR DESCRIPTION
**Pull request by colindaven**
- double trimming - first trim_galore, then trimmomatic being done at present
- test if better to use no hard? Q20 cutoff with trim_galore, but to trim twice (trim_galore THEN trimmomatic)

### CODE REVIEW

File: env.wochenende.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 name: wochenende
 channels:
-  - r
-  - conda-forge
+  - https://repo.anaconda.com/pkgs/free
   - bioconda
+  - r
   - defaults
+  - conda-forge
```
Comment (sannareddyk) [inline env.wochenende.yml:7]: @colindaven, tested latest version of Wochenende pipeline 1.7.5 with updated conda env. No issues while running the pipeline with updated env file.

File: run_Wochenende_SLURM.sh [hunk 2]
```diff
@@ -64,9 +65,9 @@ cpus=12
 
 # 2020_03 reference
 #python3 run_Wochenende.py --metagenome 2020_03_meta_human --threads $cpus --aligner bwamem --no_abra --mq30 --remove_mismatching 3 --readType SE --debug --force_restart $fastq
-#python3 run_Wochenende.py --metagenome 2020_03_meta_human --threads $cpus --aligner bwamem --nextera --no_abra --mq30 --remove_mismatching 3 --readType SE --debug --force_restart $fastq
+python3 run_Wochenende.py --metagenome 2020_03_meta_human --threads $cpus --aligner bwamem --nextera --trim_galore --no_abra --mq30 --remove_mismatching 3 --readType SE --debug --force_restart $fastq
```
Comment (sannareddyk) [inline run_Wochenende_SLURM.sh:68]: Looks good!

File: run_Wochenende.py [hunk 3]
```diff
@@ -282,7 +301,93 @@ def runAfterQC(stage_infile):
...
+        os.system(renameStr)
+
+
+    except:
+        print("Error running trim_galore")
+        sys.exit(1)
+
+    rejigFiles(stage, stage_infile, stage_outfile)
+    return stage_outfile
+
```
Comment (sannareddyk) [inline run_Wochenende.py:352]: looks fine, tested the pipeline and nextera transposase sequences are removed from 3' end.

File: run_Wochenende.py [hunk 4]
```diff
@@ -895,7 +1011,7 @@ def runBamtoolsFixed(stage_infile, numberMismatches):
         mmString
     ]
     bamtools_cmd1 = " ".join(filterMismatchesCmd)
-    print("bamtools cmd" + bamtools_cmd1)
+    print("Bamtools fixed cmd: " + bamtools_cmd1)
```
Comment (sannareddyk) [inline run_Wochenende.py:1014]: BamtoolsFixed works(mm3), reports higher read counts in output as expected when compared to mm1

**Review by sannareddyk — COMMENTED** [review]
reviewed code, tested on a sample prepped with Nextera Flex kit. Double trimming works. TBD, if Trim_galore trimming alone is enough for Nextera samples.

**Comment by colindaven** [comment]
Thanks for the thorough review Keerthi

**Pull request merged**